### PR TITLE
Fix #413 - Request profile update when updating add-on

### LINKED ISSUE #413: Blank screen on popup
Temporary fix: 

Option 1: Uninstall the add-on and reinstall it. 

Option 2: Run local commands within the developer tools: 

- Go to about:debugging, click “This Firefox” (or Nightly, etc) in the left sidebar
- Find “Firefox Relay” in the extensions list and click “Inspect”
- Click the “Console” tab
- Run the following two commands
  - Reset the timeout to check for profile updates
    - `await browser.storage.local.set({premiumCountries: {fetchedAt: 1565521770650}});`
  - Manually fetch profile updates
    - `await storeRuntimeData();`

Issue:
image

### PR DESCRIPTION
**Pull request by maxxcrawford**

### CODE REVIEW

File: src/js/background/background.js [hunk 1]
```diff
@@ -99,7 +106,8 @@ async function storeRuntimeData() {
   if (typeof existingPremiumAvailability === "object" && 
       typeof existingIntroPricingEndDate === "object" && 
       existingPremiumAvailability.fetchedAt > checkingRemainingDays &&
-      existingIntroPricingEndDate.fetchedAt > checkingRemainingDays
+      existingIntroPricingEndDate.fetchedAt > checkingRemainingDays &&
+      !opts.forceUpdate
```
Comment (maxxcrawford) [inline src/js/background/background.js:110]: This is the crux of the fix. Looking for a way to step over the date check.

File: package.json [hunk 2]
```diff
@@ -22,8 +22,8 @@
     "package:chrome": "zip -r web-ext-artifacts/chrome_relay-${npm_package_version}.zip ./src",
     "test": "echo \"Error: no test specified\" && exit 1",
     "restore-locales-github": "cd src/_locales && git restore .github/workflows/update-upstream-relay-repo.yml",
-    "remove-locales-github": "rm -r src/_locales/.github",
-    "web-ext-run": "rm -r src/_locales/.github && web-ext run -s src/",
+    "remove-locales-github": "rm -rf src/_locales/.github",
```
Comment (maxxcrawford) [inline package.json:25]: The `-f` suppresses the warning that the file has been removed when you run the `web-ext-run` command more than once locally. Added back. 

cc/ @Vinnl

File: src/js/background/background.js [hunk 3]
```diff
@@ -99,7 +106,8 @@ async function storeRuntimeData() {
   if (typeof existingPremiumAvailability === "object" && 
       typeof existingIntroPricingEndDate === "object" && 
       existingPremiumAvailability.fetchedAt > checkingRemainingDays &&
-      existingIntroPricingEndDate.fetchedAt > checkingRemainingDays
+      existingIntroPricingEndDate.fetchedAt > checkingRemainingDays &&
```
Comment (codemist) [inline src/js/background/background.js:109]: We can probably remove the `existingIntroPricingEndDate` stuff on L101, L107 and L109.

Comment (maxxcrawford) [reply src/js/background/background.js]: Removed!

**Review by codemist — APPROVED** [review]
LGTM. Does this mean bg data will get re-fetched for all users every time we do an update going forward?

**Review by Vinnl — APPROVED** [review]

**Pull request merged**